src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,114 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
In Postgres when GenerationType.AUTO is used, it generates an incremental IDs shared between the schema tables. This happens only for this database engine. To generate independent IDs for each table use GenerationType.SEQUENCE.

```suggestion
@GeneratedValue (strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")            
@SequenceGenerator(name = "BOOK_SEQ", sequenceName = "BOOK_SEQ")
```

Reply:
Strategy in GeneratedValue was changed and adding SequenceGenerator